UI/source-tree.hpp

```diff
@@ -66,6 +66,8 @@ class SourceTreeItem : public QWidget {
 	OBSSignal renameSignal;
 	OBSSignal removeSignal;
 
+	void paintEvent(QPaintEvent* event);
```

Comment:
Should be:

```cpp
	virtual void paintEvent(QPaintEvent *event) override;
```

The `virtual` technically isn't necessary but I kind of prefer it.  The `override` however is definitely always good practice I feel.